src/components/generic/TimerFunctionsUtil.tsx

```diff
@@ -0,0 +1,171 @@
+import { MutableRefObject } from "react";
+
+/* 
+ * On Pause functions
+ */
+export const onPauseStandard = (
+                status: MutableRefObject<string>, 
+                intervalRef: MutableRefObject<number | undefined>) => {
+    
+    status.current = 'pause';
+    clearInterval(intervalRef.current);
+}
+
+/* 
+ * On Start functions
+ */
+
+const onStartStandard = ( 
+    status: MutableRefObject<string>, 
+    intervalRef: MutableRefObject<number | undefined>,
+    refreshRate: number,            
+    setTime: React.Dispatch<React.SetStateAction<number>>) => {
+        
+        console.log(refreshRate);
+        if (status.current === 'start') {
+            return;
+        }
+        
+        status.current = 'start';
+        intervalRef.current = setInterval(() => {
+            setTime((time) => time + refreshRate)
+        }, refreshRate ? refreshRate < 0 ? refreshRate * -1 : refreshRate : 1000);
+}
+
+export const onStartForwardStandard = (
+                status: MutableRefObject<string>, 
+                intervalRef: MutableRefObject<number | undefined>,
+                refreshRate: number,            
+                setTime: React.Dispatch<React.SetStateAction<number>>) => {
+    
+    onStartStandard(status, intervalRef, refreshRate, setTime);
+        
+}
+
+export const onStartBackwardStandard = (
+                status: MutableRefObject<string>, 
+                intervalRef: MutableRefObject<number | undefined>,
+                refreshRate: number,            
+                setTime: React.Dispatch<React.SetStateAction<number>>) => {
+    
+    onStartStandard(status, intervalRef, refreshRate * -1, setTime);
+}
+
+export const onStartIntervalsStandard = (
```

Comment:
The intervals are off by 1000ms - they are running one interval longer than the time indicates